src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -352,7 +331,7 @@ private static Hashtable GetEditorTable(Type editorBaseType)
                 // actually run.
                 //
                 System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor(editorBaseType.TypeHandle);
-                table = s_editorTables[editorBaseType];
+                table = EditorTables[editorBaseType];
```

Comment:
If you're concerned about it, I'd be OK with it stored into a local on first access and then using the local.